### PR TITLE
Add style guide & reformat

### PR DESCRIPTION
**Pull request by jashapiro**
There is never a great time to do this, but I thought it was well past time to start using stylers in this repo when we can, and things are _kind of_ quiet at the moment, so I thought I would give it a shot. 

With that intro, here I have added a note to that effect to `CONTRIBUTING.md` and went ahead and restyled all the python, R, and rmd files. There are no code changes here aside from a couple of places where the linter wanted me to change `|` and `&` to `||` and `&&` within `if` statements, which doesn't matter for those cases but is "more correct."

There is no particular style I could find for Nextflow files, so those should be unchanged.

### CODE REVIEW

**Review by allyhawkins — APPROVED** [review]
Should we add a GHA like we have with `scpcaTools` to make sure things are styled correctly? 

Other than that, this looks good 👍

**Comment by jashapiro** [comment]
GHA would be nice, but we'd need to do a bit more to keep it up, in particular adding a python formatter. Which isn't hard, but isn't quite as canned (probably exists, but I haven't looked). We can add it as an aspirational issue.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,7 +25,17 @@ When the changes in `development` merit a new release, a pull request will be fi
 All Nextflow processes should include a [`stub` block](https://www.nextflow.io/docs/latest/process.html#stub) with a minimal script that can be run quickly to produce files in the expected output locations.
 At this stage this is purely used to allow for testing of the main workflow logic rather than the internal logic of each process.
 
-The [`test/stub-metadata.tsv`](test/stub-metadata.tsv) file is used to define input libraries that will be used for testing.
+The [`test/stub-run-metadata.tsv`](test/stub-run-metadata.tsv) file is used to define input libraries that will be used for testing.
 Any additions to the overall workflow that will allow processing of a new library type should be added into `test/stub-metadata.tsv`, along with the appropriate input files (usually empty files with the expected names) for that library type in the `test/runs/` directory.
```
Comment (sjspielman) [inline CONTRIBUTING.md:29]: This one too?
```suggestion
Any additions to the overall workflow that will allow processing of a new library type should be added into `test/stub-run-metadata.tsv`, along with the appropriate input files (usually empty files with the expected names) for that library type in the `test/runs/` directory.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,7 +25,17 @@ When the changes in `development` merit a new release, a pull request will be fi
 All Nextflow processes should include a [`stub` block](https://www.nextflow.io/docs/latest/process.html#stub) with a minimal script that can be run quickly to produce files in the expected output locations.
 At this stage this is purely used to allow for testing of the main workflow logic rather than the internal logic of each process.
 
-The [`test/stub-metadata.tsv`](test/stub-metadata.tsv) file is used to define input libraries that will be used for testing.
+The [`test/stub-run-metadata.tsv`](test/stub-run-metadata.tsv) file is used to define input libraries that will be used for testing.
```
Comment (sjspielman) [inline CONTRIBUTING.md:28]: While we're here, also should mention adding associated metadata (a row of `NA`s!) to `test/stub-sample-metadata.tsv` for new samples added.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -25,7 +25,17 @@ When the changes in `development` merit a new release, a pull request will be fi
...
 If a new reference type is needed, that should be defined in the [`test/stub-refs.json`](test/stub-refs.json) file.
 
+## Code style
+
+While there is not necessarily an established code style for nextflow code, we try to keep code neat and readable.
+Line length should generally be kept under 100 characters, and indentation should be consistent.
+
+For R code, we try to follow [`tidyverse` style conventions](https://style.tidyverse.org), and encourage the use of the [`styler`](https://styler.r-lib.org/) package to ensure that code is formatted consistently.
+
+For python code, we encourage the use of the `black` code formatter to ensure consistent formatting.
```
Comment (sjspielman) [inline CONTRIBUTING.md:39]: Add link?
```suggestion
For python code, we encourage the use of the [`black` code formatter](https://github.com/psf/black) to ensure consistent formatting.
```

Or, this link if you prefer - https://black.readthedocs.io/en/stable/

**Review by sjspielman — APPROVED** [review]
👍 left a couple small docs suggestions, but overall LGTM!

File: CONTRIBUTING.md (hunk 3 above)
Comment (jashapiro) [reply CONTRIBUTING.md:39]:
```suggestion
For python code, we encourage the use of the [`black` code formatter](https://black.readthedocs.io/en/stable/) to ensure consistent formatting.
```

**Pull request merged**